project/Build.scala

```diff
@@ -1107,7 +1107,7 @@ object Build {
 
       scalaJSExternalCompileSettings,
       inConfig(Compile)(Seq(
-          scalacOptions in doc ++= Seq("-implicits", "-groups"),
+          scalacOptions in doc ++= Seq("-implicits", "-groups", "-doc-title", "Scala.js", "-doc-version", s"${scalaJSVersion}"),
```

Comment:
This line is getting long. I suggest you split it in a meaningful way:
```scala
scalacOptions in doc ++= Seq(
    "-implicits",
    "-groups",
    "-doc-title", "Scala.js",
    "-doc-version", scalaJSVersion
),
```
(also the `s"${foo}"` is just `foo` :wink:)

(note also that we use 4 spaces of indentation when we wrap things like that).

Reply:
Done in https://github.com/scala-js/scala-js/pull/3328/commits/6ec2db62b22c552e813c2939d40d7530c2e972b3